docs/source/system_reference_guide/oss_documentation/doi_and_licensing.ipynb

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
<p>SMD (Science Mission Directorate) &lt;- Science Mission Directorate (SMD)</p><p>We don't call out uploading code to a public repository. Github, MAAP Gitlab or directly to Zenodo are acceptable (other public long term code repositories could also be acceptable, this is just a list of common options).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Maybe instead of "... made available in a publicly accessible repository that is widely recognized by the community." we could say "... made available in publicly accessible repositories such as Github, MAAP Gitlab or Zenodo." ?</p><p> </p><p> </p>

Reply:
Hmm. The only thing is I pulled these quotes and information directly from the SPD-41a, and this section is specifically stating overall guidelines for NASA-funded projects. I could change that section though. Would it work if I had a statement under the repositories section (under not NASA-funded) stating that Github and MAAP Gitlab are also acceptable for MAAP users? I do have Zenodo down there and the statement to submit a repo to the Zenodo MAAP Community

Reply:
<p>Oh, good point! Could just update that list to include those other public repositories. </p><p> </p><p>I wonder if you need the "If you are not NASA-Funded:" subheading then if some are applicable to both types of projects</p><p> </p><p>Thoughts?</p>

Reply:
overall, anything that works for NASA funded is also true for non-NASA funded, with the exception of NASA only resources. So the venn diagram is highly overlapping with a few NASA only options, mostly shared options, and then other reputable open science places for non-NASA work.

Reply:
👍 went ahead and removed the "If you are not NASA-Funded:" subheader, added some extra repos, and then made a note about Github/Gitlab for MAAP users